lib/util.js

```diff
@@ -348,9 +348,30 @@ exports.addFormatMeta = format => {
  * @param {string} html
  * @returns {string}
  */
-exports.stripHTML = html => html
+exports.stripHTML = html => decodeURIComponent(html)
   .replace(/[\n\r]/g, ' ')
   .replace(/\s*<\s*br\s*\/?\s*>\s*/gi, '\n')
   .replace(/<\s*\/\s*p\s*>\s*<\s*p[^>]*>/gi, '\n')
+  .replace(/<a\s+(?:[^>]*?\s+)?href=(?:["'])\/redirect.*?q=(.*?)(?:[&'"]).*?<\/a>/gi, '$1')
+  .replace(/<a\s+(?:[^>]*?\s+)?href=(?:["'])(https?.*?)(?:['"]).*?<\/a>/gi, '$1')
   .replace(/<.*?>/gi, '')
   .trim();
+
+/**
+ * Get a number from an abbreviated number string.
+ *
+ * @param {string} string
+ * @returns {number}
+ */
+exports.parseAbbreviatedNumber = string => {
+  string = string
+    .replace(',', '.')
+    .replace(' ', '');
+  if (string.match(/[0-9,.]*[M]/)) {
+    return parseFloat(string.replace('M', '')) * 1000000;
+  } else if (string.match(/[0-9,.]*[K]/)) {
+    return parseFloat(string.replace('K', '')) * 1000;
+  } else {
+    return parseFloat(string);
+  }
```

Comment:
would look simpler if the number + the multiplier were extracted together. something like

```js
let match = string.match(/([\d,.]+)([MK]?)/);
if (match) {
  let [num, multi] = match;
}
```